PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/startup/WACPStartUp.java

```diff
@@ -147,6 +150,10 @@ private void initialize() {
 	        }
 			EventUtil.logEvent(EventType.info, SYSTEM, PLUGIN_LOAD, null, null);
 	        AppInsightsClient.createByType(AppInsightsClient.EventType.Plugin, "", AppInsightsConstants.Load, null, true);
+	        
+	        final String machineId = TelemetryUtils.getMachieId(dataFile, Messages.prefVal, Messages.instID);
+	        final CustomerSurveyHelper helper  = new CustomerSurveyHelper(machineId, curVersion);	        
```

Comment:
We could make a static utils `getMachineId` and `getCurrentVersion` and call it in constructor instead of pass them

Reply:
It lacks dataFile

Reply:
It will lack dataFile if I move it to utility